.claude/commands/gstatus.py

```
@@ -16,184 +16,174 @@
+
         result = subprocess.run(
             cmd,
-            shell=shell,
+            shell=False,  # intentionally forced; shell param kept for API compatibility
             capture_output=capture_output,
             text=True,
             timeout=30
         )
         return result.stdout.strip() if result.returncode == 0 else None
-    except (subprocess.TimeoutExpired, subprocess.CalledProcessError, FileNotFoundError):
+    except (subprocess.TimeoutExpired, FileNotFoundError, ValueError) as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)